examples/plutus-nft/src/Cardano/PlutusExample/NFT.hs

```diff
@@ -17,35 +17,34 @@ import Data.Void (Void)
 import Ledger (
   CurrencySymbol,
   PaymentPubKeyHash,
-  Script,
   ScriptContext (scriptContextTxInfo),
   TokenName,
   TxInInfo (txInInfoOutRef),
   TxInfo (txInfoInputs, txInfoMint),
   TxOutRef,
-  mkMintingPolicyScript,
   ownCurrencySymbol,
   pubKeyHashAddress,
-  scriptCurrencySymbol,
  )
 import Ledger.Address (StakePubKeyHash)
 import Ledger.Constraints as Constraints
-import Ledger.Constraints.Metadata (
-  NftMetadata (NftMetadata),
-  NftMetadataToken (NftMetadataToken),
-  TxMetadata (TxMetadata),
-  nmtDescription,
-  nmtFiles,
-  nmtImage,
-  nmtMediaType,
-  nmtName,
-  nmtOtherFields,
- )
-import Ledger.Typed.Scripts (wrapMintingPolicy)
+
+-- import Ledger.Constraints.Metadata (
```

Comment:
Should we keep this commented?

Reply:
I think it would a better idea to remove this comment and replace it with some kind of TODO, that says add meta-data in the example.